### PR TITLE
Create new IotData client

### PR DESCRIPTION
**Pull request by tyx**

### CODE REVIEW

File: src/Integration/Symfony/Bundle/src/DependencyInjection/AwsPackagesProvider.php [hunk 1]
```diff
@@ -77,6 +77,10 @@ public static function getAllServices(): array
                 'class' => \AsyncAws\Iot\IotClient::class,
                 'package' => 'async-aws/iot',
             ],
+            'iot-data' => [
```
Comment (jderusse) [inline src/Integration/Symfony/Bundle/src/DependencyInjection/AwsPackagesProvider.php:80]: Update changelog ;)

**Comment by tyx** [comment]
psalm error seems unrelated.

However I'm not sure to get why the check of generated output is red

**Comment by jderusse** [comment]
I think you need to pull master and rebase to fix the `generated output`.
Psalm needs an exception in the baseline to be happy ;)

**Comment by tyx** [comment]
ahah and I got the same issue on my latest PR but already forget the solution, thanks ;)

**Comment by tyx** [comment]
Looks all good ! Thanks again @jderusse for your help.

**Review by jderusse — APPROVED** [review]

**Comment by jderusse** [comment]
thank you for your contribution :heart: 

The 0.1.0 version has been release https://packagist.org/packages/async-aws/iot-data

**Pull request merged**